ui/apps/platform/src/utils/entityRelationships.ts

```diff
@@ -86,6 +88,8 @@ const entityRelationshipMap: Record<string, EntityRelationshipData> = {
         // extendedMatches: [entityTypes.POLICY]
     },
     [entityTypes.NODE]: {
+        // @TODO: Uncomment this once we're using the new entity
+        // children: [entityTypes.NODE_COMPONENT],
```

Comment:
Open to suggestions on how to feature flag the following in `entityRelationships`. Since it's a separate module and outside the React lifecycle, I can't use the feature flag hook here

Reply:
As long as we feature flag the links, it may not be worth feature-flagging the direct "sub-routes" built from this map. (Re-factoring this map everywhere it is used would be a risky change.)